PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -208,7 +212,7 @@ struct HfTrackIndexSkimCreatorTagSelCollisions {
 
   /// Collision selection
   /// \param collision  collision table with
-  template <bool applyTrigSel, bool applyCentSel, typename Col>
+  template <bool applyTrigSel, bool applyCentSel, int centEstimator = CentralityEstimators::None, typename Col>
```

Comment:
In general, it increases readability when all non-type template arguments are provided explicitly.
Moreover here, the `None` case does not seem needed.

Reply:
Good point, it is indeed non necessary / redundant. Indeed I thought that it was better to remove the `applyCentSel` argument and rely on the `centEstimartor` only (in case of `None` now no centrality selection is applied).